lib/dal/src/func/authoring/save.rs

```diff
@@ -18,13 +30,14 @@ pub(crate) async fn save_func(
     description: Option<String>,
     code: Option<String>,
     associations: Option<FuncAssociations>,
-) -> FuncAuthoringResult<(SavedFunc, Func)> {
+) -> FuncAuthoringResult<SavedFunc> {
     let func = Func::get_by_id_or_error(ctx, id).await?;
 
-    // Disallow the ability to modify builtins.
-    if func.builtin {
-        return Err(FuncAuthoringError::NotWritable);
-    }
+    // TODO(nick): we should eventually either return an error or make it a no-op.
+    // For now, we need configurable builtins to ensure that the system is working.
+    // if func.builtin {
+    //     return Err(FuncAuthoringError::NotWritable);
+    // }
```

Comment:
Temporary measure. It wasn't working as intended and broke the UI. Not only that, we are using it for testing func authoring. This is mainly for the user to not break their environments. We will re-enable this once things settle.